src/specs/networking.md

```diff
@@ -66,3 +68,26 @@ Accepting a `WireTxPayForMessage` into the mempool requires different logic than
+| `root`        | [HashDigest](./data_structures.md#hashdigest)                                               | The Merkle root of the offending row or column.                                                                      |
+| `proof`       | [NamespaceMerkleTreeInclusionProof](./data_structures.md#namespacemerkletreeinclusionproof) | The Merkle proof of the row or column root in [`availableDataRoot`](./data_structures.md#header).|
+| `isCol`       | `bool`                                                                                      | A Boolean indicating if it is an offending row or column; `false` if it is a row.                                    |
+| `position`    | `uint64`                                                                                    | The index of the row or column in the square.                                                                        |
+
+
+## StateFraudProof
+
+| name                       | type                                                                                     | description                                                                                                                                                                                                                                                                                  |
+|----------------------------|------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `headerForTxns`            | [Header](./data_structures.md#header)                                                    | Header containing `availableDataRoot`(./data_structures.md#header) of type [HashDigest](./data_structures.md#hashdigest) committing to transaction data.                                                                                              |
+| `headerForISRs`            | [Header](./data_structures.md#header)                                                    | Header containing `availableDataRoot`(./data_structures.md#header) of type [HashDigest](./data_structures.md#hashdigest) committing to intermediate state root data. `lastHeaderHash` within `headerForISRs` equals the hash of `headerForTxns`.|
+| `transactionShareProofs`   | [ShareProof](./data_structures.md#shareproof)`[]`                                        | ShareProof contains both the [Share](./data_structures.md#share) and [NamespaceMerkleTreeInclusionProof](./data_structures.md#namespacemerkletreeinclusionproof) for the Share. `isCol` of type `bool` is set to `false`.                                           |
```

Comment:
Describing the contents of `ShareProof` here is redundant, since the data structure is already defined elsewhere. Also, clarify that `isCol` _must_ be `false`.

Reply:
Edited.